spyder_kernels/comms/commbase.py

```diff
@@ -455,7 +455,7 @@ def _async_error(self, error_wrapper):
         """
         Handle an error that was raised on the other side asyncronously.
         """
-        error_wrapper.raise_error()
+        error_wrapper.print_error()
```

Comment:
Why this change?

Reply:
I don't want to raise the error for async call, because if the call is async, the caller will not be expecting an exception. This is only a problem on the kernel side as the frontend reimplements the function.

Reply:
For example, if I make a syncronous call, I don't want to have an error for an unrelated async call being raised.

Reply:
Thinking about it, I have the following question:
The kernel makes a call that fails on the frontend side. Where should the error go?
 - If the error is syncronous, it should go back to the kernel side, because the calling function might want to handle the error.
 - If the call is asyncronous, the kernel side should be notified as well, so the callback for this call can be discarded.

But should we send back async error to the frontend to be displayed in the error window?

Reply:
I can do that easily. Should I open a new PR or attach this to this one?

Reply:
> But should we send back async error to the frontend to be displayed in the error window?

Yes, I think that's important to do it.

> Should I open a new PR or attach this to this one?

Please do it here (this PR is already very simple).